ms-patches/reduce-allocation-merges.yml

```diff
@@ -11,4 +11,12 @@ title: Reduce Allocation Merges
   - Add a new Class to support the rematerialization of SR objects that are part of a merge;
   - Patch HotSpot to be able to serialize and deserialize debug information related to allocation merges;
   - Patch C2 to generate unique types for SR objects participating in some allocation merges.
+  - Fix: Incorrect rematerialization of scalar replaced objects in C2
```

Comment:
Why doesn't this note mention openjdk/jdk#17562 like the jdk11u PR?
Same question for jdk17u

Reply:
That was a "typo" in JDK11 PR. I just copied the titles of the PRs with the fixes and it ended copying that part as well.